src/DataTable/index.jsx

```diff
@@ -191,7 +191,7 @@ DataTable.propTypes = {
     /** User visible column name */
     Header: PropTypes.oneOfType([PropTypes.func, PropTypes.node]).isRequired,
     /** String used to access the correct cell data for this column */
-    accessor: PropTypes.string.isRequired,
+    accessor: PropTypes.string,
```

Comment:
this prop is not required for expander column (when you want to render expandable rows), this also caused console errors